testonly/hammer/hammer.go

```diff
@@ -181,15 +185,58 @@ func HitMap(cfg MapConfig) error {
 		}
 	}(ticker.C)
 
-	for count := uint64(1); count < cfg.Operations; count++ {
-		if err := s.retryOneOp(ctx); err != nil {
-			return err
+	var wg sync.WaitGroup
+	errs := make(chan error, cfg.NumCheckers)
```

Comment:
Is buffering the err channel needed?

Reply:
Yes, I think so, otherwise the channel blocks the checker goroutines from terminating (and adding the `errs`-draining loop needs to be after the `wg.Wait()` to get all the errors).

Reply:
Blocking the checkers from terminating should be ok, no?  Is anything waiting on them?